### PR TITLE
HW Develop Cleanup

### PR DESCRIPTION
**Pull request by kbrimm**
Adjusts file structure and naming for clarity, rounds up some miscellaneous files that were floating around in email (node_test.ino). Updates readme with new library inclusion.

### CODE REVIEW

**Comment by ghost** [comment]
Looks okay to me, but I would like Steven to back me up on that.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
+  * https://github.com/matthijskooijman/arduino-lmic
+* **Radiohead Library** 
+  * http://lowpowerlab.com/RadioHead_LowPowerLab.zip
+* **DHT Sensor Library**
+  * https://github.com/adafruit/DHT-sensor-library
+
+Plug the micro controller into the computer, and set the device address at line 48. When compiled, the program will be pushed to the micro-controller and execute immediately!
+
+## Note:
+
```
Comment (stevengoo) [inline README.md:34]: Maybe note the FINAL FINAL version for our NODE code is now called node controller?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+Get your hardware:
+* **Micro controller** (Motenino USB RFM95 with LORA 915mhz Trasnsceiver)
+
+Install the Arduino IDE:
+* **Arduino IDE** (version 1.6.9 or greater)
+  * https://www.arduino.cc/en/Main/Software
+* Under tools tab, make sure the board is "Arduino/Genuino Uno".
+  
+Install the following in your Arduino/libraries directory:
+* **IMIC Master Library**
```
Comment (stevengoo) [inline README.md:20]: comment this library is for the communication between hub and node

File: README.md [hunk 3]
```diff
@@ -1,10 +1,32 @@
...
+
+Install the Arduino IDE:
+* **Arduino IDE** (version 1.6.9 or greater)
+  * https://www.arduino.cc/en/Main/Software
+* Under tools tab, make sure the board is "Arduino/Genuino Uno".
+  
+Install the following in your Arduino/libraries directory:
+* **IMIC Master Library** 
+  * https://github.com/matthijskooijman/arduino-lmic
+* **Radiohead Library**
```
Comment (stevengoo) [inline README.md]: comment library is for testing LORA communication between nodes. (basically not needed for our project, can skip if don't want to test LORA card on motenios)

File: README.md [hunk 4]
```diff
@@ -1,10 +1,32 @@
...
+* **Arduino IDE** (version 1.6.9 or greater)
+  * https://www.arduino.cc/en/Main/Software
+* Under tools tab, make sure the board is "Arduino/Genuino Uno".
+  
+Install the following in your Arduino/libraries directory:
+* **IMIC Master Library** 
+  * https://github.com/matthijskooijman/arduino-lmic
+* **Radiohead Library** 
+  * http://lowpowerlab.com/RadioHead_LowPowerLab.zip
+* **DHT Sensor Library**
```
Comment (stevengoo) [inline README.md:23]: comment library allows to read data from our DHT22 sensor. That is our temperature and humidity sensor!

**Comment by stevengoo** [comment]
I added a few suggestions to the readme file to help understand why we added those libraries! Also I have a more updated node controller file! It includes the code to read the data from all the sensors, but I haven't posted it up yet because it hasn't been tested yet due to the hardware issues talked about few days ago. 

Should I post that up before Bart looks at this? Also should I include a paragraph of what we mean by data loss?

I believe its normal for this data loss to occur because the connection of node to hub isn't direct. The hub knows to look for the node, but the node doesn't know who is listening. It is still secured since the hub needs to have the right key. Since it isn't a direct connection, I believe our only solution is to have the nodes not send once every 5 minutes but a couple of times every five minutes. Then we parse that information, grab the newest one for each node and send that batch to the DB. Sapphire suggest some type of dictionary system for this.

File: README.md (hunk 1 above)
Comment (kbrimm) [inline README.md:34]: Good catch. It totally didn't occur to me that this would change the note information!

**Comment by kbrimm** [comment]
I trust your evaluation of the data loss - I'm comfortable with what you suggest. I'll merge these changes now and check with Bart tomorrow just to be sure there's not something we're overlooking.

**Pull request merged**